internal/exec/util/user_lookup.go

```diff
@@ -48,3 +48,26 @@ func (u Util) userLookup(name string) (*user.User, error) {
 
 	return usr, nil
 }
+
+// groupLookup looks up the group in u.DestDir.
+func (u Util) groupLookup(name string) (*user.Group, error) {
```

Comment:
Does it make sense to rename this file if we are doing both user and group lookup?